### DISCUSSION TITLE
Display server errors

### DISCUSSION THREAD

**Discussion by mylastore** [itswadesh/svelte-commerce#23] [Q&A]
**Describe the bug**
Adding items to the cart feels like the app is not working with no warning been displayed. 

**To Reproduce**
I downloaded the repo and install dependencies and try adding an item to the cart. 

**Expected behavior**
There should be server errors warnings. I see the error on the web console "401 unauthorized" invalid token...

**Comment by itswadesh** [comment]
Is the cart quantity notification increasing on addToCart?

**Comment by mylastore** [comment]
No increase in the cart

**Comment by itswadesh** [comment]
Issue is fixed with the new version
